site/2017/04/01/announcing-doctrine-4.rst

```diff
@@ -0,0 +1,97 @@
+Announcing Doctrine 4
+=====================
+
+It has been few months since we shed some light on the `future of Doctrine project <https://github.com/doctrine/doctrine2/issues/6211>`__
+and had an extensive insight into what `Guilherme <https://twitter.com/guilhermeblanco>`__ had been up to. Adding
+tens of megabytes of IRC logs of internal discussion we felt we owe you an update on where Doctrine is and where
+it's heading to:
+
+Doctrine 3 is no more!
+----------------------
+
+Rest assured, we are in no way ditching all the code Guilherme hacked so far or any of the ideas that sprung
+for the next major release. We are still looking into leveraging all goodies that were given to us with PHP 7.
+We want the next Doctrine version to be an extremely stable and realiable piece of software. We are also still trying to figure out how to maintain all projects under Doctrine's umbrella effectively. To recap,
+Doctrine is not only the ORM you all know, we are also maintaining a number of ODM projects (`MongoDB <https://github.com/doctrine/mongodb-odm>`__
+or `CouchDB <https://github.com/doctrine/couchdb-odm>`__ to name few) which all share basic concepts and code.
+Some of them also face a major rewrite, like the much anticipated MongoDB ODM 2.0 with support for new MongoDB driver.
+
+Joining Forces
+--------------
+
+Instead of having each team iterate on its own, implementing the same concepts multiple times across various libraries,
+we decided it's for the best if we all work on one project and make it as good and robust as possible.
+We decided it's time to break boundaries so Doctrine 4 will be all about interoperability. **Doctrine 4 will support
+both RDBMS and NoSQL databases at the same time!**
+
+Following latest trends
+-----------------------
+
+A really big thing we want to (re)introduce is an Active Record pattern. We've ran a poll on the #doctrine IRC
+channel and it turned out that 68% of developers are dying a little bit each time they are injecting a service which
+barely saves data to a database and miss simplicity of having an entity save itself.

```

Comment:
Since the developers "miss simplicity of having...", I'd wrap "which barely saves data to a database" in commas as a clause. Also, "miss the simplicity" would read better.